pkg/provider/aws/resources/ecr_test.go

```diff
@@ -119,6 +118,8 @@ func Test_ImageCreate(t *testing.T) {
 				return
 			}
 			tt.want.Assert(t, dag)
+			graphImage := dag.GetResourceByVertexId(image.Id().String())
```

Comment:
nit: inconsistent where to use `ByVertexId` and when to use the regular one (taking in the `ResourceId`)